docs/install.rst

```diff
@@ -171,7 +176,7 @@ The typical annotations for a web application used via a web browser are:
 
    annotations:
     kubernetes.io/ingress.class: nginx
-    nginx.ingress.kubernetes.io/auth-request-redirect: $request_uri
+    nginx.ingress.kubernetes.io/auth-method: GET
```

Comment:
why is this one changing?

Reply:
Yeah, sorry, this is somewhat unrelated and I didn't update the PR overview. It turns out that the `auth-request-redirect` that we were setting is just the default and thus doesn't do anything, so removing it makes everything shorter. And the `auth-method` setting does two things: first, if the attempt is a `POST`, apparently it would have sent a `POST` auth request, which would break everything. This overrides that. And second it will force NGINX to set `X-Original-URI`, which we wanted for logging.

Reply:
is the change to auth-method something we need to apply to all the ingress rules?

Reply:
Yup, it is. I'll make another PR for lsp-deploy to set the `auth-method` annotation.